.github/workflows/hw-ci/run_hw_ci.sh

```diff
@@ -4,6 +4,7 @@
 # .github/workflows/hw-ci/run_hw_ci.sh com.rusefi.HwCiProteus
 
 HW_SUITE=$1
+SIMULATOR_INI_FILE=$2
```

Comment:
how is this new SIMULATOR_INI_FILE used in run_hw_ci.sh?

Reply:
Sorry I didn't commit it, I'll fix it with rebase now, it's used in `java_console/io/src/main/java/com/rusefi/LocalIniFileProvider.java ` to override the hardcoded variable `(INI_FILE_FOR_SIMULATOR_ROOT_PATH)` for the ini file